config.json

```diff
@@ -204,6 +204,15 @@
         "difficulty": 3,
         "status": "active"
       },
+      {
+        "slug": "gigasecond",
+        "name": "Gigasecond",
+        "uuid": "6a6aca53-2dd8-4673-9f72-893eeaadd368",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 4,
```

Comment:
We may need to revisit difficulties.

Reply:
Lower?

Reply:
Eh. It requires reading the time functions. But beyond that it's not too bad. I'd do 3.

Reply:
Fair enough